dubbo-common/src/main/java/org/apache/dubbo/common/utils/StringUtils.java

```diff
@@ -508,6 +508,14 @@ public static boolean isContains(String values, String value) {
         return isNotEmpty(values) && isContains(COMMA_SPLIT_PATTERN.split(values), value);
     }
 
+    public static boolean isContains(String str, char ch) {
+        return isNotEmpty(str) && str.indexOf(ch) >= 0;
+    }
+
+    public static boolean isNotContains(String str, char ch) {

```

Comment:
Can be implemented by the above method.

Reply:
I think what satansk suggests is quite reasonable, could you please take a look ?

Reply:
OK，I will rewrite it